src/nyx/python/nyxus/nyxus_arrow.py

```diff
@@ -0,0 +1,28 @@
+import os
+import sys
+
+def link_arrow_lib():
+    
+    if arrow_headers_found():
+        import pyarrow as pa
+    
+    if os.sys.platform == "win32":
```

Comment:
If `arrow_headers_found()` returns `False`, this will result an error. So, we should execute it only for `True` case